app/views/foreman_ansible/job_templates/package_action_-_ansible_default.erb

```diff
@@ -34,26 +34,36 @@ model: JobTemplate
 # For Windows targets use the win_package module instead.
 ---
 - hosts: all
+  vars:
+    pkg_name: "<%= input('name') %>"
   <%- if input('pre_script').present? -%>
   pre_tasks:
     - shell: "<%= input('pre_script') %>"
   <%- end -%>
   tasks:
+    - name: Validate input
+      ansible.builtin.fail:
+        msg: "The 'name' input cannot be empty. A package name must be provided."
+      when: pkg_name is not defined or pkg_name | length == 0
```

Comment:
Maybe it would be worth to make this assert instead of fail so that users don't see `[skipped] Validate input` when the input name is valid. Feel free to fiddle with the messages, these are just an example.

```suggestion
    - name: Validate input
      ansible.builtin.assert:
        that: pkg_name is defined and pkg_name | length != 0
        fail_msg: "The 'name' input cannot be empty. A package name must be provided."
        success_msg: "Valid 'name' input: '{{ pkg_name }}'"
```